index.js

```diff
@@ -23,8 +24,11 @@ class App extends MatrixPuppetBridgeBase {
 
     this.client.on('message', (ev) => {
       const { source, message } = ev.data;
+      let room = source;
+      if ( message.group != null)
```

Comment:
This isn't a blocker, but you may want to use brackets for single-statement conditions even though it's syntactically valid without them, because if someone doesn't notice that the brackets are missing, they may add code they think is inside the conditional block, to be surprised when it always executes, or to have it go unnoticed..
```suggestion
      if ( message.group != null) {
```